<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QOpenGLWidget" name="canvas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxTools">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>338</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Ferramentas</string>
    </property>
    <layout class="QGridLayout" name="gridTools">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnQuad">
       <property name="text">
        <string>Quadrado</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnRectangle">
       <property name="text">
        <string>Retângulo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnLine">
       <property name="text">
        <string>Linha</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnCurve">
       <property name="text">
        <string>Curva</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="boxRotation">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>176</width>
      <height>56</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotação</string>
    </property>
    <layout class="QGridLayout" name="gridRotation">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnLeft">
       <property name="text">
        <string>Esquerda</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnRight">
       <property name="text">
        <string>Direita</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="boxOptions">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>460</y>
      <width>257</width>
      <height>56</height>
     </rect>
    </property>
    <property name="title">
     <string>Opções</string>
    </property>
    <layout class="QGridLayout" name="gridOptions">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Limpar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnOpen">
       <property name="text">
        <string>Abrir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
